STCommon/src/commonMain/kotlin/com/infomaniak/multiplatform_swisstransfer/common/matomo/MatomoCategory.kt

```diff
@@ -0,0 +1,35 @@
+/*
+ * Infomaniak SwissTransfer - Multiplatform
+ * Copyright (C) 2024 Infomaniak Network SA
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+package com.infomaniak.multiplatform_swisstransfer.common.matomo
+
+enum class MatomoCategory(val categoryName: String) {
```

Comment:
Is it possible to "extend" this enum class in `appleMain` or `androidMain`? So we could add categories limited to a specific OS? @KevinBoulongne / @FabianDevel